guides/micronaut-scope-types/micronaut-scope-types.adoc

```diff
@@ -47,9 +48,13 @@ callout:singleton[1]
 
 === Singleton Test
 
+To use the testing features described in this section, add the following dependency to your build file:
```

Comment:
Please, move this to a `common:` snippet.

Reply:
Just did, please review again, thanks